<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgQueryInfo</class>
 <widget class="QDialog" name="QDlgQueryInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查看客户</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainGridLayout"/>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout7">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>车牌号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtPlateID">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>发动机号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtMotorID">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>品牌</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtBrand">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>车系</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtSeries">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>购车日</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtBuyDate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>行驶公里数</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edtOdometer">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout5">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>手机号码</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtMobilePhone">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>固定号码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtLandline">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>邮箱地址</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtEMail">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>客户地址</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtAddress">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>所属区域</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtDistrict">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>证件号码</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edtCredentialsID">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_12">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout8">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>车辆颜色</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtColor">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>年检日期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtAannualSurveyDate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>商业险到期日</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtCommercialInsuranceExpiration">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>交强险到期日</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtMandatoryInsuranceExpiration">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>历史来店次数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtVistCount">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>保险公司</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edtInsurer">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout4">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>客户编号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtCustomerID">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>客户名称</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtName">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>客户类别</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtCustomerCategory">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>隶属部门</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtDepartment">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>客户专员</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtAccountExecutive">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>客户来源</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edtCustomerSource">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout6">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>初次来访日</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtVistFirstTime">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>建档日期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtEntryTime">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>最近一次到店时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtVistNewlyTime">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>会员卡类别</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtMembershipClass">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>会员卡余额或次数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtMembershipCardSurplus">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>历史消费金额</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edtConsumptionAmount">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_13">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout9">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>车辆类型</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edtFeature">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>首保日期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtMaintenanceFirstDate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>下次保养日期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtMaintenanceNextDate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>车辆总体状况</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtState">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 192, 192);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout2">
    <item row="0" column="0">
     <widget class="QLabel" name="lblVehicleBasic">
      <property name="text">
       <string>车辆基本信息</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="lineVehicle">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout3">
    <item row="0" column="0">
     <widget class="QLabel" name="lblServiceRecord">
      <property name="text">
       <string>服务记录</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="lineService">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout1">
    <item row="0" column="0">
     <widget class="QLabel" name="lblCustomerBasic">
      <property name="text">
       <string>客户基本信息</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="lineCustomer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget5" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget6" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget7" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget8" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget9" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDetail">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>详细信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget10" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>841</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="lblSpace">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>600</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tabServiceRecord">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtCustomerID</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtCustomerCategory</tabstop>
  <tabstop>edtDepartment</tabstop>
  <tabstop>edtAccountExecutive</tabstop>
  <tabstop>edtCustomerSource</tabstop>
  <tabstop>edtMobilePhone</tabstop>
  <tabstop>edtLandline</tabstop>
  <tabstop>edtEMail</tabstop>
  <tabstop>edtAddress</tabstop>
  <tabstop>edtDistrict</tabstop>
  <tabstop>edtCredentialsID</tabstop>
  <tabstop>edtVistFirstTime</tabstop>
  <tabstop>edtEntryTime</tabstop>
  <tabstop>edtVistNewlyTime</tabstop>
  <tabstop>edtMembershipClass</tabstop>
  <tabstop>edtMembershipCardSurplus</tabstop>
  <tabstop>edtConsumptionAmount</tabstop>
  <tabstop>edtPlateID</tabstop>
  <tabstop>edtMotorID</tabstop>
  <tabstop>edtBrand</tabstop>
  <tabstop>edtSeries</tabstop>
  <tabstop>edtBuyDate</tabstop>
  <tabstop>edtOdometer</tabstop>
  <tabstop>edtColor</tabstop>
  <tabstop>edtAannualSurveyDate</tabstop>
  <tabstop>edtCommercialInsuranceExpiration</tabstop>
  <tabstop>edtMandatoryInsuranceExpiration</tabstop>
  <tabstop>edtVistCount</tabstop>
  <tabstop>edtInsurer</tabstop>
  <tabstop>edtFeature</tabstop>
  <tabstop>edtMaintenanceFirstDate</tabstop>
  <tabstop>edtMaintenanceNextDate</tabstop>
  <tabstop>edtState</tabstop>
  <tabstop>tabServiceRecord</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
